projects/fsastorefrontlib/src/cms-components/checkout/components/bind-quote-dialog/bind-quote-dialog.component.ts

```diff
@@ -40,26 +45,54 @@ export class BindQuoteDialogComponent {
 
   bindQuote() {
     this.quoteService.bindQuote(this.cartCode);
-    combineLatest([this.cartService.getActive(), this.cartService.isStable()])
+    combineLatest([
```

Comment:
Shouldn't this be wrapped in `this.subscription.add`, since you call subscribe method.

Reply:
Yes it should.